sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -3655,6 +3656,109 @@ def shift(self, axis, freq, **kwargs):
   describe = _agg_method(pd.DataFrame, 'describe')
   max = _agg_method(pd.DataFrame, 'max')
   min = _agg_method(pd.DataFrame, 'min')
+
+  @frame_base.with_docs_from(pd.DataFrame)
+  @frame_base.args_to_kwargs(pd.DataFrame)
+  @frame_base.populate_defaults(pd.DataFrame)
+  def pivot(self, index=None, columns=None, values=None, **kwargs):
```

Comment:
Oh please also add a docstring here indicating that any columns specified in `columns` must have CategoricalDType so we can determine the output column names. There's docstring infrastructure that will put this message in the "Differences from pandas" section, e.g. for align: 
![image](https://user-images.githubusercontent.com/675055/160923353-f75e81b7-ea4b-48ee-8567-c78016799cdc.png)

Reply:
Done